kordac/KordacExtension.py

```diff
@@ -34,30 +33,26 @@ def __init__(self, tags=[], html_templates={}, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def extendMarkdown(self, md, md_globals):
-        processors = {
-            'preprocessors': {
-                'save-title': ['save-title', SaveTitlePreprocessor(self, md), '_begin'],
-                'remove-title': ['remove-title', RemoveTitlePreprocessor(self, md), '_end'],
-                #'commentpre': ['commentpre', CommentPreprocessor(self, md), '_begin'],
-                },
-            'blockprocessors': {
-                #'heading': ['hashheader', NumberedHashHeaderProcessor(self, md.parser), '_begin'],
-                #'panel': ['panel', PanelBlockProcessor(self, md.parser), '>ulist'],
-                #'glossary-link': ['glossary-link', GlossaryLinkBlockProcessor(self, md.parser), '_begin'],
-                #'interactive': ['interactive', InteractiveBlockProcessor(self, md.parser), '_begin'],
-                #'video': ['video', VideoBlockProcessor(self, md.parser), '_begin'],
-                #'image': ['image', ImageBlockProcessor(self, md.parser), '_begin'],
-                #'comment': ['comment', CommentBlockProcessor(self, md.parser), '>ulist'],
-                #'button-link': ['button-link', ButtonLinkBlockProcessor(self, md.parser), '_begin']
-                },
-            }
+        preprocessors = [
+            ['comment', CommentPreprocessor(self, md), '_begin'],
+            ['save-title', SaveTitlePreprocessor(self, md), '>comment'],
```

Comment:
I encountered an issue when I implemented `remove-title` that if I had it's order as `>save-title` and `save-title` wasn't present, it through an error. However since you have changed the structure to an ordered list, you should be able to state `comment` is always at the start.

Would it be correct to have `comment` at the end of the preprocessor list, so it's added last to the first position?